### DISCUSSION TITLE
*Suggestion* Aquaculture neptunium hoe compatability

### DISCUSSION THREAD

**Discussion by The-Lone-Potato** [vectorwing/FarmersDelight#949] [Ideas]
The aquaculture neptunium hoe works by converting vanilla dirt into an aquaculture variant of normal farmland that is permanently moisturised. Could an unobtainable rich soil farmland variant also be added to allow the neptunium hoe to permanently moisturise rich soil farmland please?

**Comment by The-Lone-Potato** [comment]
Could this change, if it is made, be done to the 1.20.1 version of the mod?

**Comment by vectorwing** [comment]
Moved the issue over to the Discussions tab, as this is where suggestions should be posted. 👍

Hmm... I'm not sure if it's worthwhile to add a specific block here just for Aquaculture's sake. This kind of compat may make more sense from Aquaculture's end, depending on how they implement their own perma-hydrated farmland.

**Reply by The-Lone-Potato** [reply]
Firstly, sorry for not posting the suggestion here first. I'm not really familiar with the layout of Github.
Secondly, I did pose this on the end of aquaculture, which was how I found out about how the hoe converts soil to a seperate always hydrated block. I was told that they 'weren't planning' on implementing the block, probably because then they might have to do the same to a bunch of other modded farmlands. Given that I think I'm the only person who's ever had this problem, and I'm suggesting it for 1.20.1, it's difficult for me to say with a straight face that this is an important, glaring issue (it AINT).
I appereciate the response though, thank you and I love the mod!
